devito/function.py

```diff
@@ -950,6 +950,11 @@ def __init__(self, *args, **kwargs):
             else:
                 raise TypeError("`space_order` must be int or 3-tuple of ints")
 
+            # Symbolic (finite difference) coefficients
+            self._symbolic_coefficients = kwargs.get('coefficients', 'standard')
```

Comment:
nitpicking: I'd call it just "coefficients", as ``_symbolic_coefficients`` is a bit too long for an instance attribute

Reply:
Yes, will change this - it's not actually a good name considering the arguments it takes. Originally, when I named it in another branch it made more sense (prior to some changes).